cmds/ln/ln.go

```diff
@@ -45,11 +47,11 @@ type config struct {
 // promptOverwrite ask for overwrite destination
 func promptOverwrite(fname string) bool {
 	fmt.Printf("ln: overwrite '%v'? ", fname)
-	var txt string
-	if fmt.Scanln(&txt); txt != "y" {
-		return false
+	answer, err := bufio.NewReader(os.Stdin).ReadString('\n')
+	if err == nil && strings.ToLower(answer)[0] == 'y' {
```

Comment:
Just

```go
return err == nil && strings.ToLower(answer)[0] == 'y'
```

Reply:
ACK. will push that soon. You have been quicker with your last changes...